opta/opta_run_check.py

```diff
@@ -0,0 +1,41 @@
+"""
+Usage: This file is used to check if Opta is already running in the current directory.
+This creates a file in opta directory with the name: .opta_running_<underscored-cwd>_<pid>
+"""
+
+
+import glob
+import os
+
+from opta.constants import HOME
+from opta.exceptions import UserErrors
+
+OPTA_RUNNING_LOC_PREFIX = os.path.join(
+    HOME, ".opta", f".opta_running_{os.getcwd().replace(os.path.sep, '_')}_"
```

Comment:
why not use the current working directory when you run opta?
this is usually how these files are located, ex: https://www.terraform.io/language/files/dependency-lock#lock-file-location

for us it could be `$CWD/.opta.lock`

Reply:
Not using the `CWD`, just to keep things clean in user's directory.
No other reason for it.